EventFilter/SiPixelRawToDigi/plugins/SiPixelRawToDigi.cc

```diff
@@ -97,6 +98,11 @@ SiPixelRawToDigi::SiPixelRawToDigi( const edm::ParameterSet& conf )
     usePhase1 = config_.getParameter<bool> ("UsePhase1");
     if(usePhase1) edm::LogInfo("SiPixelRawToDigi")  << " Use pilot blade data (FED 40)";
   }
+  //CablingMap could have a label //Tav
+  CablingMapLabel = ""; 
+  if (config_.exists("CablingMapLabel")) {
```

Comment:
this "if" is not necessary after the value is added to the ::fillDescriptions method